pygmt/src/grdclip.py

```diff
@@ -83,8 +83,8 @@ def grdclip(grid, **kwargs):
     Example
     -------
     >>> import pygmt
-    >>> # Load a grid of @earth_relief_30m data, with an x-range of 10 to 30,
-    >>> # and a y-range of 15 to 25
+    >>> # Load a grid of @earth_relief_30m data, with a longitude range of
```

Comment:
Actually, I'm not sure if we should have the `°` symbol in the examples. It's a non-ASCII symbol.

Reply:
This is how the `grdhisteq` docstrings look like if opened with "GB 2312" encoding. It looks pretty bad.
![Screenshot from 2023-10-12 16-10-14](https://github.com/GenericMappingTools/pygmt/assets/3974108/eda55331-9ab5-4fd3-bd84-621e8baf11dc)

GB 2312 is the default encoding for the Windows Chinese version, but I'm not sure about the default encoding when VSCode opens a plaintext file.

Reply:
Yeah, that looks not good, and we should change this to either not using the `°` symbol or writing `deg`.

Reply:
I prefer to "10 E", instead of "10 deg E".

Reply:
I have updated this in commit https://github.com/GenericMappingTools/pygmt/pull/2740/commits/88d4b751ea011945b44c966f71e0e4ba45351605.

Reply:
Sorry, after thinking twice, I think I've changed my mind. These non-ASCII characters are rendered correctly on the PyGMT site, so they won't affect most users.

Users who open PyGMT source codes in their text editor should know that UTF-8 is always the best and default encoding for Python codes (actually for any codes).

I feel we should revert the changes in 88d4b751ea011945b44c966f71e0e4ba45351605.

Reply:
No worries, that's fine with me. I reverted commit https://github.com/GenericMappingTools/pygmt/commit/88d4b751ea011945b44c966f71e0e4ba45351605 in commit https://github.com/GenericMappingTools/pygmt/pull/2740/commits/df55aeebff1fff2bb0f028d3ce00f55b7d9cff9b.